components/post_view/post/post.jsx

```diff
@@ -234,7 +252,15 @@ class Post extends React.PureComponent {
     }
 
     unsetHover = () => {
-        this.setState({hover: false});
+        this.setState({hover: false, alt: false});
+    }
+
+    handleAlt = (e) => {
```

Comment:
duplicate function?

Reply:
Oh yeah, I noticed that at some point, but forgot to fix it. I'll fix it once that other PR is merged